crates/hyperqueue/src/client/commands/job.rs

```diff
@@ -102,6 +118,35 @@ pub async fn output_job_detail(
     Ok(())
 }
 
+pub async fn output_job_cat(
+    gsettings: &GlobalSettings,
+    connection: &mut ClientConnection,
+    job_selector: Selector,
+    task_selector: Selector,
+    stdout: bool,
+    stderr: bool,
+) -> anyhow::Result<()> {
+    let message = FromClientMessage::JobDetail(JobDetailRequest {
+        selector: job_selector,
+        include_tasks: true,
+    });
+    let responses =
+        rpc_call!(connection, message, ToClientMessage::JobDetailResponse(r) => r).await?;
+
+    for response in responses {
+        if let Some(job) = response.1 {
+            gsettings.printer().print_job_cat(
+                job,
+                stdout,
+                stderr,
+            );
+        } else {
+            log::error!("Job {} not found", response.0);
```

Comment:
This should return an error instead of just logging. Once only a single job is supported, it should be easy.